### DISCUSSION TITLE
batch converter adding extra 10 seconds

### DISCUSSION THREAD

**Discussion by nammminn2** [SubtitleEdit/subtitleedit#6380] [Q&A]
i didnt click any of the options[https://dsc.cloud/4cd306/Screen-Recording-2022-11-03-05-03-16]( https://dsc.cloud/4cd306/Screen-Recording-2022-11-03-05-03-16)

**Comment by niksedk** [comment]
Could you attach the files?
They probably have a header, that suggests starting after 10 seconds...

It's possible to turn off using this header:
![image](https://user-images.githubusercontent.com/1261331/199889470-61ebe1b5-ca8a-4598-8f69-a29fe8873f3b.png)
